compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -232,6 +239,11 @@ def AMDAIEDistributeL1Allocations :
   let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIEDistributeL1AllocationsPass()";
 }
 
+def AMDAIEDmaBdChain :
```

Comment:
For consistency with how other transformations are named, add a verb to the pass name, like `AMDAIECreateDmaBdChain(s)` or `AMDAIEInsertDmaBdChain(s)` for example.